tokio/src/runtime/runtime.rs

```diff
@@ -119,6 +119,7 @@ pub enum RuntimeFlavor {
     MultiThread,
     /// The flavor that executes tasks across multiple threads.
     #[cfg(tokio_unstable)]
+    #[cfg_attr(docsrs, doc(cfg(tokio_unstable)))]
```

Comment:
This was not mentioned in the related issue, but I did a check for all the tokio_unstable marked items across the repo and this one was the only one that was obviously missing.